go.mod

```diff
@@ -405,7 +405,7 @@ replace (
 	github.com/naoina/toml => github.com/naoina/toml v0.1.1
 	github.com/nbutton23/zxcvbn-go => github.com/nbutton23/zxcvbn-go v0.0.0-20171102151520-eafdab6b0663
 	github.com/olekukonko/tablewriter => github.com/olekukonko/tablewriter v0.0.0-20170122224234-a0225b3f23b5
-	github.com/onsi/ginkgo => github.com/onsi/ginkgo v1.10.1
+	github.com/onsi/ginkgo => github.com/onsi/ginkgo v1.10.4-0.20191206104214-f66e896ca772
```

Comment:
can we split out the ginkgo bump commit?

Reply:
You mean into another PR? Sure. Doing it now.

Reply:
Done!